pkg/chains/storage/grafeas/grafeas.go

```diff
@@ -240,13 +260,14 @@ func (b *Backend) createNote(ctx context.Context, tr *v1beta1.TaskRun, opts conf
 //   - its intoto payload and signature will be stored in a BUILD occurrences for each image artifact generated from the taskrun
 //   - each BUILD occurrence will have the same data but differ in the ResourceUri field
 //   - the identifier/ResourceUri is IMAGE_URL@IMAGE_DIGEST
-func (b *Backend) createOccurrence(ctx context.Context, tr *v1beta1.TaskRun, payload []byte, signature string, opts config.StorageOpts) ([]*pb.Occurrence, error) {
+func (b *Backend) createOccurrence(ctx context.Context, obj objects.TektonObject, payload []byte, signature string, opts config.StorageOpts) ([]*pb.Occurrence, error) {
 	occs := []*pb.Occurrence{}
 
 	// create Occurrence_Attestation for OCI
 	if opts.PayloadFormat == formats.PayloadTypeSimpleSigning {
-		uri := b.retrieveSingleOCIURI(tr, opts)
-		occ, err := b.createAttestationOccurrence(ctx, tr, payload, signature, uri)
+		uri := b.retrieveSingleOCIURI(obj, opts)
+		b.logger.Info("chuangw6 - uri for att occ ", uri)
```

Comment:
Is this a debugging comment that was accidentally left in?

Reply:
Yes, thanks @bcaton85!